src/main/resources/schemas/connections_config.json

```diff
@@ -33,6 +33,10 @@
               "type": "string",
               "title": "JDBC driver class name"
             },
+            "max_num_retries": {
+              "type": "integer",
+              "title": "The number of times a query can be retried"
```

Comment:
Could we include the default value for this property (as we do for connection type)?